frontend/src/metabase/query_builder/components/expressions/Editor/Listbox/Listbox.module.css

```diff
@@ -27,6 +27,8 @@
 
 .label {
   white-space: pre;
+  overflow: hidden;
```

Comment:
I'm not sure if ellipsis is a proper fix. There is no tooltip, so it's not possible to distinguish these 2 names:
- "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt 1"
- "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt 2"

In v52 the dropdown grew horizontally: [screenshot](https://github.com/user-attachments/assets/b811ac5d-c006-4ebe-8c7a-81067fc947b3)

I suggest to double-check the expected result with the product team